bot/cogs/team.py

```diff
@@ -0,0 +1,344 @@
+    def __init__(self, bot):
+        self.bot = bot
+        self.teams = []
+        logger.info("Team cog initialized")
+        self.clean_up_vc.start()
+
+    def create_team(self, team, name):
+        i = 2
+        new_name = name
+        while new_name in [t.name for t in self.teams]:
+            new_name = name + "." + str(i)
+            i += 1
+        teamObj = Team(new_name)
+        self.teams.append(teamObj)
+
+        for member in team:
+            for t in self.teams:
+                for m in t.members:
+                    if member.id == m.id:
+                        t.rem_member(member)
+            teamObj.add_member(member)
+
+    @commands.group(
+        name="team", invoke_without_command=True, help="Get help for team commands"
+    )
+    async def team(self, ctx):
+        await reply_and_delete(ctx, "'team' needs a sub-command!")
+        await ctx.send_help(self.team)
+
+    @team.command(help="Create a new team using: !team create [name(s)]")
+    @commands.has_permissions(administrator=True)
+    async def create(self, ctx, *args):
+        for name in args:
+            existing_t = False
+            for t in self.teams:
+                if t.name == name:
+                    existing_t = True
+                    await reply_and_delete(ctx, "Team " + name + " already exists.")
+            if not existing_t:
+                self.teams.append(Team(name))
+                await reply_and_delete(ctx, "Team " + name + " was added!")
+
+    @team.command(help="Remove a team using: !team remove [name(s)]")
+    @commands.has_permissions(administrator=True)
+    async def remove(self, ctx, *args):
+        for name in args:
+            team_found = False
+            for t in self.teams:
+                if t.name == name:
+                    team_found = True
+                    try:
+                        if t.vc:
+                            await t.vc.delete()
+                        self.teams.remove(t)
+                        await reply_and_delete(ctx, "Team " + name + " was removed.")
+                    except:
+                        await reply_and_delete(
+                            ctx, "Could not remove team " + name + "."
+                        )
+            if not team_found:
+                await reply_and_delete(
+                    ctx, "Could not remove team " + name + ", does it exist?"
+                )
+
+    @team.command(help="Rename team using: !team rename [from] [to]")
+    @commands.has_permissions(administrator=True)
+    async def rename(self, ctx, old: str, new: str):
+        for t in self.teams:
+            if t.name == old:
+                t.name = new
+                return
+        await reply_and_delete(ctx, "Could not find team " + old)
+
+    @team.command(help="Show list of teams")
+    async def list(self, ctx):
+        t_list = ""
+        for t in self.teams:
+            t_list += t.name + ", "
+        await reply_and_delete(ctx, t_list[:-2])
```

Comment:
Change to:
`await reply_and_delete(ctx, if t_list[:-2] else "There are no teams.")`

Currently the bot won't respond if there are no teams, and an error will occur.